.expeditor/end_to_end.pipeline.yml

```diff
@@ -112,3 +112,15 @@ steps:
           environment:
             - BUILD_PKG_TARGET=x86_64-linux
             - HAB_BLDR_URL=$ACCEPTANCE_HAB_BLDR_URL
+  - label: "[:linux: test_studio_auto_installs]"
```

Comment:
This is a personal preference, but I find these pipeline stage definitions to be easier to read if they're separated from each other by newlines.

Reply:
Agreed,  that was a copy/paste error on my part.  Fixed :)